parlai/core/torch_agent.py

```diff
@@ -501,6 +609,18 @@ def add_cmdline_args(
             ' Typically, scripts can set their preferred default behavior at the start,'
             ' e.g. eval scripts.',
         )
+        agent.add_argument(
+            '--record-silence',
+            type='bool',
+            default=True,
+            help='Whether to record silence token in the agent history when the agent self-observes. Defaults to True for backward-compatability.',
+        )
+        agent.add_argument(
+            '--silence-token',
```

Comment:
1. could we default this to `'__SILENCE__'`? It's the only one we ever use
2. could you specify in the help that this is a string, just to differentiate from e.g. tokenized strings (which are ints)